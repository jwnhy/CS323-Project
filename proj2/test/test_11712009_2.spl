int main(int a, int b) {
    b = a;
    c = 1;
}