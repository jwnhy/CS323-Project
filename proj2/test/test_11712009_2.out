Error type 1 at Line 3: Variable not found in scope.
Error type 5 at Line 3: Two sides of assignment are not of the same type.
